configs/hrnet/README.md

```diff
@@ -0,0 +1,55 @@
+# High-resolution networks (HRNets) for object detection
+
+## Introduction
+
+```
+@inproceedings{SunXLW19,
+  title={Deep High-Resolution Representation Learning for Human Pose Estimation},
+  author={Ke Sun and Bin Xiao and Dong Liu and Jingdong Wang},
+  booktitle={CVPR},
+  year={2019}
+}
+
+@article{SunZJCXLMWLW19,
+  title={High-Resolution Representations for Labeling Pixels and Regions},
+  author={Ke Sun and Yang Zhao and Borui Jiang and Tianheng Cheng and Bin Xiao 
+  and Dong Liu and Yadong Mu and Xinggang Wang and Wenyu Liu and Jingdong Wang},
+  journal   = {CoRR},
+  volume    = {abs/1904.04514},
+  year={2019}
+}
+```
+
+## Results and Models
+
+Faster R-CNN
+
+| Backbone|#Params|GFLOPs|Lr sched|mAP|Download|
+| :--:|:--:|:--:|:--:|:--:|:--:|
+| HRNetV2-W18 |26.2M|159.1| 1x | 36.1 | [model](https://1drv.ms/u/s!Avk3cZ0cr1Jeake7zkUDn0zJE8w)|
+| HRNetV2-W18 |26.2M|159.1| 2x | 38.1 | [model](https://1drv.ms/u/s!Avk3cZ0cr1JeaeLG5k4Ic7WzHHc)|
```

Comment:
[20, 23, 24] is not the standard 2x ([16, 22, 24]) schedule, which will be higher. Here we need to denote the schedule as `20-23-24e`.